go/networkqualityd.go

```diff
@@ -49,6 +49,14 @@ func init() {
 	}
 }
 
+// setCors makes it possible for wasm clients to connect to the server
+// from a webclient that is not hosted on the same domain.
+func setCors(h http.Header) {
+	h.Set("Access-Control-Allow-Origin", "*")
+	h.Set("Access-Control-Allow-Headers", "*")
+	h.Set("Content-Type", "application/octet-stream")
```

Comment:
can this be moved outside of this function and applied to the responses that don't have a content-type? (eg the config already has json CT)

Reply:
Great point! Sorry!

Reply:
Hope this looks better!